lib/express-rate-limit.js

```diff
@@ -27,52 +53,41 @@ function RateLimit(options) {
         }
     });
 
+    options = defaults(options, {
```

Comment:
Ok, one idea, and this is a small thing, but I think an improvement. When you call defaults and pass an object with anew MemoryStore instance, that MemoryStore is created weather it ends up being use or not. So if the user passes in a custom store, the default MemoryStore will still be created and use up a (tiny) bit of resources.

If you did this instead, it would only create a MemoryStore instance when the user didn't specify any options.store:

    options.store = options.store || new MemoryStore(options);

This works because 

1. JS lets you use the logical or (`||`) on objects, returning the first "truthy" one (meaning that it exists) and 
2. it uses "short circuit evaluation" - if the left side of `||` is truthy, the the right side doesn't even get evaluated, so no new MemoryStore instance is created.